draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1254,10 +1254,10 @@ attribute-name of HttpOnly and an empty attribute-value.
 If the attribute-name case-insensitively matches the string "SameSite", the
 user agent MUST process the cookie-av as follows:
 
-1.  Let `enforcement` be "None".
+1.  Let `enforcement` be "Default".
 
-2.  If cookie-av's attribute-value is a case-insensitive match for "Strict",
-    set `enforcement` to "Strict".
+2.  If cookie-av's attribute-value is a case-insensitive match for "None",
+    set `enforcement` to "None".
 
```

Comment:
We lost "Strict" here! I think you need to insert the "None" step and renumber the subsequent steps, rather than replacing the existing step 2.

Reply:
Whoops. 😅  Nice catch!